packages/composites/crypto-volatility-index/test/integration/adapter.test.ts

```diff
@@ -2,28 +2,32 @@ import { Requester } from '@chainlink/ea-bootstrap'
 import { assertError, assertSuccess } from '@chainlink/ea-test-helpers'
 import { AdapterRequest } from '@chainlink/types'
 import { execute } from '../../src/adapter'
+jest.setTimeout(10000)
 
 describe('execute', () => {
   const jobID = '1'
-  const contractAddress = '0x0BD102ef50a6a133B38Bf3Bd3d40cE36cc1aB5A8'
+  const contractAddress = '0x1B58B67B2b2Df71b4b0fb6691271E83A0fa36aC5'
+  process.env.COINGECKO_DATA_PROVIDER_URL = 'http://localhost:3000'
```

Comment:
We should not hard-code this env var, instead we should have it set when running the test.

```suggestion
```